packages/pds-ember/ember-cli-build.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+
+module.exports = function(defaults) {
+  let app = new EmberAddon(defaults, {
+    // See https://github.com/adopted-ember-addons/ember-cli-sass
+    sassOptions: {
+      // TBD...
+    },
+    'ember-cli-storybook': {
+      enableAddonDocsIntegration: true,
+      componentFilePathPatterns: [
+        'addon/components/**/*.js',
+      ],
+    },
+    hinting: false,
+    svgJar: {
+      optimizer: false, // prevents optimizing pre-optimized icons
+      sourceDirs: [
+        './tests/dummy/public',
+        // NOTE: This probably won't work with yarn 2 .yarn/cache/,
+        //       because there's no node-modules/ directory.
+        '../../node_modules/@hashicorp/structure-icons/dist',
+      ],
+    },
+  });
+
+  /*
+    This build file specifies the options for the dummy test app of this
+    addon, located in `/tests/dummy`
+    This build file does *not* influence how the addon or the app using it
+    behave. You most likely want to be modifying `./index.js` or app's build file
+  */
+
+  return app.toTree();
+};
```

Comment:
Majority of the configuration in this file is pulled directly from the addon in HCP.